skopt/optimizer/optimizer.py

```diff
@@ -172,7 +173,32 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
                 "Expected acq_optimizer to be 'lbfgs' or 'sampling', "
                 "got %s" % acq_optimizer)
 
-    def ask(self):
+    def ask(self, n_points=None, strategy="CL"):
+        """Query point or multiple points to evaluate.
+
+        * `n_jobs` [int or None]:
```

Comment:
and we should update the text describing what this does. I'll think about a suggestions.